files/webfile_test.go

```diff
@@ -0,0 +1,38 @@
+package files
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"testing"
+)
+
+func TestWebFile(t *testing.T) {
+	http.HandleFunc("/my/url/content.txt", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "Hello world!")
+	})
+	listener, err := net.Listen("tcp", ":18281")
```

Comment:
It would probably be cleaner to use https://godoc.org/net/http/httptest#NewServer for this, in case there already is something listening on that port.

Reply:
ah yeah, you're totally right, I always forget.